readme.md

```diff
@@ -1,10 +1,9 @@
 # Search bar for Airbnb
 Replace `<your_account>` with your Github username and copy the links to Pull Request description:
-- [DEMO LINK](https://<your_account>.github.io/layout_search-bar-airbnb/)
-- [TEST REPORT LINK](https://<your_account>.github.io/layout_search-bar-airbnb/report/html_report/)
+- [DEMO LINK](https://vchasnyk.github.io/layout_html-form/)
+- [TEST REPORT LINK](https://vchasnyk.github.io/layout_html-form/report/html_report/)
```

Comment:
There are links for another task. Please add the correct links and edit the PR description